scripts/jenkins/ardana/ansible/repositories.yml

```diff
@@ -62,10 +42,10 @@
         src: SLES12-SP3-Pool
       - name: SLES12-SP3-Updates
         src:  SLES12-SP3-Updates
-      - name: SUSE-OpenStack-Cloud-8-Pool
-        src: SUSE-OpenStack-Cloud-8-Pool
-      - name: SUSE-OpenStack-Cloud-8-Updates
-        src: SUSE-OpenStack-Cloud-8-Updates
+        # - name: SUSE-OpenStack-Cloud-8-Pool
+        # src: SUSE-OpenStack-Cloud-8-Pool
+        # - name: SUSE-OpenStack-Cloud-8-Updates
+        # src: SUSE-OpenStack-Cloud-8-Updates
```

Comment:
Why remove these? They still need to be mounted in order for ardana to pick them up. Although I guess they're not really needed as long as the development media is there...

Also, why comment these out instead of removing them?

Reply:
exactly, the logic is that we need to avoid them being available for the case that we use the devel media (to make sure that *only* the packages from the devel media are used, not some othe rpackage that accidentally has a higher version number in the pool/update channel).

I can remove that if you think commenting out is bad for some reason.